recipes/hla-la/1.0/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.0" %}
+
+package:
+  name: hla-la
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/DiltheyLab/HLA-LA/archive/v{{ version }}.tar.gz
+  sha256: 68e1398537c09b4ef5ac00eca10343e5a0ede70a29b2cfc9c4feefffa5bde57c
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} 
+    - jsoncpp
```

Comment:
jsoncpp can probably be removed from build

Reply:
This is correct, thank you!